xbmc/PlayListPlayer.h

```diff
@@ -103,6 +103,11 @@ class CPlayListPlayer : public IMsgTargetCallback
    */
   int GetCurrentPlaylist() const;
 
+  /*! \brief Get currently active playlist or a default for current player
```

Comment:
> obviously not. if you can't put it easily into words, it is not good.

I put it in in words just before, but I understand the risk of that code ending up being dead. I'll change it.